engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -34,6 +38,8 @@ class EngineJobExecutionActor(jobKey: BackendJobDescriptorKey,
   val jobTag = s"${workflowId.shortString}:${jobKey.call.fullyQualifiedName}:${jobKey.index.fromIndex}:${jobKey.attempt}"
   val tag = s"EJEA_$jobTag"
 
+  private var callCachingActorProps: Props = _
```

Comment:
I'm not a big fan of these types of initializations but as long as we keep them to a dull roar and clearly mark them as dangerous (but why it isn't in this case) I'm ok w/ it.